bool request
---
geometry_msgs/TransformStamped left_transform
geometry_msgs/TransformStamped right_transform